backend/repositories/match-repository.js

```diff
@@ -72,13 +73,26 @@ const constructMatch = async (matchDescription) => {
     return new Match(teams, matchDescription.team1Won, date, ratingChange)
 }
 
+const getElapsedSecondsSinceLastMatch = async () => {
+    const matchTimes = (await storage.getAllMatches()).map(match => match.date.getTime()) 
```

Comment:
all matches? 😱

Reply:
Is good enough :smile

Reply:
How hard would it be to implement getting only the last match from the DB?

Reply:
One SQL query, not interested in writing those though. I kinda like ORM 😄

Reply:
Ok, fixed that! Please re-review